felix/docker-image/Dockerfile

```diff
@@ -51,24 +52,33 @@ RUN apt-get update && apt-get install -y \
     # Felix FV tests require these
     wget \
     ethtool \
-    tcpdump \
-    hping3
+    hping3 \
+    libpcap0.8 \
+    tcpdump
 
-# Copy in wg tool from stage 1.
-COPY --from=wgtool /usr/bin/wg /usr/bin/wg
+RUN apt-get install --no-install-recommends -y wireguard-tools
+
+RUN apt-get clean
 
 # Default to the xtables backend of iptables.
 RUN update-alternatives --set iptables /usr/sbin/iptables-legacy
 RUN update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
 
 # Felix is built with RHEL/UBI and links against libpcap.so.1. We need this symbolic link
 # until Debian changes the soname from .0.8 to .1.
-RUN ln -s libpcap.so.1.8.1 /usr/lib/x86_64-linux-gnu/libpcap.so.1
+RUN ln -s $(readlink /usr/lib/*-linux-gnu/libpcap.so.0.8) $(ldconfig -v 2>/dev/null | grep /usr/lib/ | head -n 1 | sed -e 's/://')/libpcap.so.1
```

Comment:
`x86_64-linux-gen` is not correct for all arches. The updated script still creates the same symlink but read the architecture specific string from `ldconfig` output.